2024-05-29 17:06:01|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|apple
2024-05-29 17:06:01|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:01|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:01|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|aaaa
2024-05-29 17:06:01|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:01|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:01|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|bbbb
2024-05-29 17:06:01|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:01|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|cccc
2024-05-29 17:06:01|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|dddd
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|eeee
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|ffff
2024-05-29 17:06:01|   DEBUG|testAction.py[:57]|hhhh
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|gggg
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|iiii
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|jjjj
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|kkkk
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|llll
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|oooo
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|   DEBUG|testAction.py[:57]|pppp
2024-05-29 17:06:02|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:02|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:02|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:06:03|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:06:03|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|admin
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|sniper
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:19|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:19|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|sniper
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|111111
2024-05-29 17:10:19|   ERROR|testAction.py[:73]|用户名或密码错误
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 67, in dologin
    raise RuntimeError("用户名或密码错误")
RuntimeError: 用户名或密码错误
2024-05-29 17:10:19|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|apple
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:19|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:19|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|aaaa
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:19|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:19|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|bbbb
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:19|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:19|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:19|   DEBUG|testAction.py[:57]|cccc
2024-05-29 17:10:19|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|dddd
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|eeee
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|ffff
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|hhhh
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|gggg
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|iiii
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|jjjj
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|kkkk
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:20|   DEBUG|testAction.py[:57]|llll
2024-05-29 17:10:20|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:20|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:21|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:21|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:21|   DEBUG|testAction.py[:57]|oooo
2024-05-29 17:10:21|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:21|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:21|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:21|   DEBUG|testAction.py[:57]|pppp
2024-05-29 17:10:21|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:10:21|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:21|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:10:21|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:10:21|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:18:22|   DEBUG|testAction.py[:57]|admin
2024-05-29 17:18:22|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:18:22|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:18:22|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:18:27|   DEBUG|testAction.py[:218]|1
2024-05-29 17:18:27|   DEBUG|testAction.py[:219]|10
2024-05-29 17:18:27|   DEBUG|testAction.py[:223]|0
2024-05-29 17:18:27|   DEBUG|testAction.py[:224]|10
2024-05-29 17:32:48|   DEBUG|testAction.py[:57]|admin
2024-05-29 17:32:48|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:32:48|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:32:48|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:35:56|   DEBUG|testAction.py[:57]|admin
2024-05-29 17:35:56|   DEBUG|testAction.py[:58]|123456
2024-05-29 17:35:56|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-29 17:35:56|    INFO|testAction.py[:105]|载入主页面...
2024-05-29 17:35:58|   DEBUG|testAction.py[:218]|1
2024-05-29 17:35:58|   DEBUG|testAction.py[:219]|10
2024-05-29 17:35:58|   DEBUG|testAction.py[:223]|0
2024-05-29 17:35:58|   DEBUG|testAction.py[:224]|10
2024-05-30 14:15:19|   DEBUG|testAction.py[:218]|1
2024-05-30 14:15:19|   DEBUG|testAction.py[:219]|10
2024-05-30 14:15:19|   DEBUG|testAction.py[:223]|0
2024-05-30 14:15:19|   DEBUG|testAction.py[:224]|10
2024-05-30 14:15:29|   ERROR|testAction.py[:95]|(1062, "Duplicate entry 'admin' for key 'username'")
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\mysql\base.py", line 73, in execute
    return self.cursor.execute(query, args)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\cursors.py", line 163, in execute
    result = self._query(query)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\cursors.py", line 321, in _query
    conn.query(q)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 505, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 724, in _read_query_result
    result.read()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 1069, in read
    first_packet = self.connection._read_packet()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 676, in _read_packet
    packet.raise_for_error()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\protocol.py", line 223, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.IntegrityError: (1062, "Duplicate entry 'admin' for key 'username'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 91, in addUser
    User.objects.create_superuser(username='admin', password='123456', email='123456@qq.com')
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\contrib\auth\models.py", line 163, in create_superuser
    return self._create_user(username, email, password, **extra_fields)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\contrib\auth\models.py", line 146, in _create_user
    user.save(using=self._db)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\contrib\auth\base_user.py", line 67, in save
    super().save(*args, **kwargs)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\base.py", line 740, in save
    force_update=force_update, update_fields=update_fields)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\base.py", line 778, in save_base
    force_update, using, update_fields,
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\base.py", line 881, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\base.py", line 921, in _do_insert
    using=using, raw=raw,
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\query.py", line 1270, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\models\sql\compiler.py", line 1416, in execute_sql
    cursor.execute(sql, params)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 98, in execute
    return super().execute(sql, params)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\django\db\backends\mysql\base.py", line 73, in execute
    return self.cursor.execute(query, args)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\cursors.py", line 163, in execute
    result = self._query(query)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\cursors.py", line 321, in _query
    conn.query(q)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 505, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 724, in _read_query_result
    result.read()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 1069, in read
    first_packet = self.connection._read_packet()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\connections.py", line 676, in _read_packet
    packet.raise_for_error()
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\protocol.py", line 223, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "D:\pyCharm\pycharm\testProject\venv\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.IntegrityError: (1062, "Duplicate entry 'admin' for key 'username'")
2024-05-30 14:15:29|    INFO|testAction.py[:99]|response json: {'code': -1, 'msg': '(1062, "Duplicate entry \'admin\' for key \'username\'")'}
2024-05-30 14:18:27|   DEBUG|testAction.py[:218]|1
2024-05-30 14:18:27|   DEBUG|testAction.py[:219]|10
2024-05-30 14:18:28|   DEBUG|testAction.py[:223]|0
2024-05-30 14:18:28|   DEBUG|testAction.py[:224]|10
2024-05-30 14:18:30|   DEBUG|testAction.py[:218]|2
2024-05-30 14:18:30|   DEBUG|testAction.py[:219]|10
2024-05-30 14:18:30|   DEBUG|testAction.py[:223]|10
2024-05-30 14:18:30|   DEBUG|testAction.py[:224]|20
2024-05-30 14:18:31|   DEBUG|testAction.py[:218]|3
2024-05-30 14:18:31|   DEBUG|testAction.py[:219]|10
2024-05-30 14:18:31|   DEBUG|testAction.py[:223]|20
2024-05-30 14:18:31|   DEBUG|testAction.py[:224]|30
2024-05-30 14:18:32|   DEBUG|testAction.py[:218]|17
2024-05-30 14:18:32|   DEBUG|testAction.py[:219]|10
2024-05-30 14:18:32|   DEBUG|testAction.py[:223]|160
2024-05-30 14:18:32|   DEBUG|testAction.py[:224]|170
2024-05-30 15:00:15|   DEBUG|testAction.py[:218]|1
2024-05-30 15:00:15|   DEBUG|testAction.py[:219]|10
2024-05-30 15:00:15|   DEBUG|testAction.py[:223]|0
2024-05-30 15:00:15|   DEBUG|testAction.py[:224]|10
2024-05-30 15:13:30|   DEBUG|testAction.py[:57]|admin
2024-05-30 15:13:30|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:30|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:13:30|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:30|   DEBUG|testAction.py[:57]|sniper
2024-05-30 15:13:30|   DEBUG|testAction.py[:58]|111111
2024-05-30 15:13:30|   DEBUG|testAction.py[:57]|apple
2024-05-30 15:13:30|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:30|   ERROR|testAction.py[:73]|用户名或密码错误
Traceback (most recent call last):
  File "D:\pyCharm\pycharm\testProject\testProject\testAction.py", line 67, in dologin
    raise RuntimeError("用户名或密码错误")
RuntimeError: 用户名或密码错误
2024-05-30 15:13:30|    INFO|testAction.py[:78]|response json: {'code': 11, 'msg': '用户名或密码错误'}
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|aaaa
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|bbbb
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|cccc
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|dddd
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|eeee
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
2024-05-30 15:13:31|    INFO|testAction.py[:78]|response json: {'code': 0}
2024-05-30 15:13:31|    INFO|testAction.py[:105]|载入主页面...
2024-05-30 15:13:31|   DEBUG|testAction.py[:57]|ffff
2024-05-30 15:13:31|   DEBUG|testAction.py[:58]|123456
